client/src/Components/FeelingsLog/FeelingsLog.css

```diff
@@ -24,6 +25,49 @@ th {
     text-align: left;
 }
 
-.feelingsLogs tbody tr:nth-of-type(odd) {
-    background-color: var(--background-color);
-}
\ No newline at end of file
+.truncate {
+    max-width: 450px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+
+table thead tr {
+    display: block;
+}
+
+table th,
+table td { 
+    width: 100px; 
+    min-width: 100px; 
+}
+
+table tbody {
+    display: block;
+    overflow-y: scroll;
+    width: 100%;
+    height: 650px;
+}
+
+.activity-head {
+    width: 450px;
+}
+
+.feeling-head {
+    width: 110px;
+}
+
+tbody::-webkit-scrollbar {
+    width: 7px;
+  }
+  
+  tbody::-webkit-scrollbar-track {
+    background-color: var(--element-bkgd-color); 
+  }
+  
+  tbody::-webkit-scrollbar-thumb {
+    background-color: var(--element-bkgd-color);
+    border-radius: 20px;
+    border: solid rgba(255,255,255, .5);
+  }
```

Comment:
Excited to see how this styling looks! Love that you added some scrollbar styling.

Reply:
Thank you for working on this Amber!

Reply:
+1 - loved the sneak peek we got!